scripts.py

```diff
@@ -153,20 +153,38 @@ class IdentifierInputScript(Script):
     """A script that takes identifiers as command line inputs."""
 
     @classmethod
-    def parse_command_line(cls, _db=None, cmd_args=None, *args, **kwargs):
+    def read_stdin_lines(self, stdin):
```

Comment:
I had about 6000 identifiers that needed reindexing, way too many to put on the command line, so I changed IdentifierInputScript so that you can pass those identifiers in on standard input. This was tricky because a) it's impossible to test the 'real' standard input, and b) if you run the script without providing anything on standard input, I don't want the script to hang waiting for your input.

This method is not tested but I have tested it manually.